src/model.ts

```diff
@@ -366,7 +369,8 @@ export function build<TSchema extends BaseSchema, TOptions extends SchemaOptions
     async function bulkWrite(
       operations: PaprBulkWriteOperation<TSchema, TOptions>[],
       options?: BulkWriteOptions
-    ): Promise<BulkWriteResult> {
+    ): Promise<BulkWriteResult | void> {
+      if (operations.length === 0) return;
```

Comment:
```suggestion
      if (operations.length === 0) {
      	return;
      }

```

Reply:
Let's add a unit test for this case.

Reply:
We can assert that the underlying `bulkWrite` collection method isn't being called - I'll add that now.